specification/elastic/resource-manager/Microsoft.Elastic/preview/2023-02-01-preview/elastic.json

```diff
@@ -1962,6 +2002,17 @@
         }
       }
     },
+    "ElasticVersionsListResponse": {
```

Comment:
>"ElasticVersionsListResponse": { [](http://example.com/codeflow?start=4&length=32)

Schema should have a description or title.

Reply:
Will add.